Source/Core/VideoBackends/D3D/TextureCache.cpp

```diff
@@ -29,6 +30,42 @@ static std::unique_ptr<PSTextureEncoder> g_encoder;
 const size_t MAX_COPY_BUFFERS = 32;
 ID3D11Buffer* efbcopycbuf[MAX_COPY_BUFFERS] = {0};
 
+static u32 GetLevelPitch(HostTextureFormat format, u32 row_length)
```

Comment:
I guess this isn't D3D related, so this function should be somewhere in common. Maybe close to HostTextureFormat?